src/RecordManager/Base/Record/Marc.php

```diff
@@ -634,7 +635,7 @@ public function getLinkingIDs()
             $results = [...$results, ...$cns];
         }
 
-        return $results;
+        return array_unique(array_filter($results));
```

Comment:
Actually, no need for array_filter either, I believe.
```suggestion
        return $results;
```